main.go

```diff
@@ -159,14 +159,17 @@ func registerRHSM(ctx *cli.Context) (string, error) {
 					}
 
 					// Ask for organization and display hint with list of organizations
-					fmt.Printf("Available Organizations:\n")
-					for _, org := range orgs {
-						fmt.Printf(" - %v\n", org)
-					}
-					fmt.Printf("\n")
-
 					scanner := bufio.NewScanner(os.Stdin)
-					fmt.Print("Organization: ")
+					fmt.Println("Available Organizations:")
+					writer := tabwriter.NewWriter(os.Stdout, 0, 2, 2, ' ', 0)
+					for i, org := range orgs {
+						fmt.Fprintf(writer, "%v\t", org)
```

Comment:
The `fmt.Fprintf()` returns `int` and `err`. I suppose that we do not care about error in this case, then please use:

```go
_, _ = fmt.Fprintf(writer, "%v\t", org)
```